### PR TITLE
User can see articles based on their location 

### PR DESCRIPTION
**Pull request by Carltesio**
# [PT Story](https://www.pivotaltracker.com/story/show/172048790)
## Changes proposed in this pull request:
* Adds Geo location stub to test the feature
* Adds Location item on the header component

## What I have learned working on this feature:
* Stubbing geo location to test within cypress
* Refreshed knowledge about implementing and testing Devise Token Auth

## Screenshots (Only if client side story)

### CODE REVIEW

File: cypress/support/stubLocation.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            }
+        };
+        cy.stub(win.navigator.geolocation, "getCurrentPosition").callsFake(
+            callback => {
+                return callback(stubLocation);
+            }
+        )
+    }
+});
+
```
Comment (emtalen) [inline cypress/support/stubLocation.js:16]: Format the intendent spaces, should be 2 not 4

**Review by emtalen — COMMENTED** [review]
Just the formatting, otherwise 💯 , and also look into why semaphore is not working 🙈

File: src/App.jsx [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
 import DisplayComponents from "./components/DisplayComponents";
+import axios from 'axios'
 
 const App = props => {
+  const dispatch = useDispatch()
+  useEffect(() => {
+    navigator.geolocation.getCurrentPosition(async pos => {
+      const currentSession = await axios.post('https://newsroom3api.herokuapp.com/api/v1/auth/sign_in', { location: pos.coords })
+      dispatch({ type: "SET_CURRENT_SESSION", payload: currentSession.data })
+      debugger;
```
Comment (pgaunitz) [inline src/App.jsx]: remove debugger

**Review by emtalen — COMMENTED** [review]
Just remove the debugger

**Review by pgaunitz — APPROVED** [review]
remove debugger
LG

**Pull request merged**